admin/client.py

```diff
@@ -279,6 +296,8 @@ def get_steps_pkg(distribution, package_source=PackageSource()):
     package_manager = DOCKER_IMAGES[distribution].package_manager
     steps = [
         ensure_minimal_setup(package_manager),
+        allow_notty_sudo(distribution),
+        install_distro_package("curl", distribution),
```

Comment:
It's possible this should go in the ensure_minimal_setup but I'm unsure if we should pollute that function with installing curl.